### PR TITLE
Display ANS NFT

### PR DESCRIPTION
**Pull request by h0ngcha0**
ANS NFTs use std number `000301`, which is not handled by SDK and explorer backend well. 

We can fix SDK and explorer backend for `000301` after its standardized. 

- Collection page:
image

- NFT page:
image

### CODE REVIEW

File: packages/extension/src/ui/features/accountNfts/alephium-nft.service.ts [hunk 1]
```diff
@@ -19,6 +19,22 @@ export const fetchCollectionAndNfts = async (
       parentAndTokenIds[collectionId] = [tokenId]
     }
   }
+
+  // TODO: Fix explorer backend for 000301 NFTs
+  if (nftMetadataz.length != nonFungibleTokenIds.length) {
```
Comment (Lbqds) [inline packages/extension/src/ui/features/accountNfts/alephium-nft.service.ts:24]: Question: If NFT `000301` is supported in the explorer backend, do we still need this?

Comment (h0ngcha0) [reply packages/extension/src/ui/features/accountNfts/alephium-nft.service.ts:24]: No, we don't need this after the explorer backend is fixed

**Review by Lbqds — APPROVED** [review]
LGTM 👍

File: packages/extension/src/ui/features/accountTokens/tokens.state.ts [hunk 2]
```diff
@@ -136,7 +136,7 @@ export const useNonFungibleTokensWithBalance = (
       const nonFungibleTokens: BaseTokenWithBalance[] = []
       for (const token of potentialNonFungibleTokens) {
         if (nonFungibleTokens.findIndex((t) => t.id == token.id) === -1) {
-          const tokenType = await fetchImmutable(`${token.id}-token-type`, () => nodeProvider.guessStdTokenType(token.id))
+          const tokenType = await fetchImmutable(`${token.id}-token-type`, () => nodeProvider.guessStdInterfaceId(token.id))
```
Comment (polarker) [inline packages/extension/src/ui/features/accountTokens/tokens.state.ts:139]: With `1.2.2`, can we still use `guessStdTokenType`?

Comment (h0ngcha0) [reply packages/extension/src/ui/features/accountTokens/tokens.state.ts:139]: Yes we can, we should use `guessStdTokenType` here.

**Review by polarker — APPROVED** [review]
LGTM 👍

**Pull request merged**